app/views/content/steps-to-become-a-teacher/_step_2_check_your_qualifications.html.erb

```diff
@@ -10,12 +10,12 @@
 
 <p>You may be able to show you meet the standard in another way if you do not have GCSEs.</p>
 
-<p>For advice on qualifications, chat to our teacher training experts online. They're available Monday to Friday between 8:30am and 5:30pm.</p>
+<p>For advice on the qualifications you need for teaching, chat to our teacher training experts online. They're available Monday to Friday between 8:30am and 5:30pm.</p>
```

Comment:
```suggestion
<p>For advice on the qualifications you need for teacher training, chat to our experts online. They're available Monday to Friday between 8:30am and 5:30pm.</p>
```
change to teacher training rather than teaching, as this is mainly what this section is talking about?